### DISCUSSION TITLE
using await

### DISCUSSION THREAD

**Discussion by mikasoukhov** [dotnet/csharplang#5393] [Language Ideas] [answered]
Now

`using var _ = await ....`

Can be

`using await ...`

**Comment by svick** [comment, marked as answer]
This is a duplicate of https://github.com/dotnet/csharplang/discussions/2593.

**Reply by DavidArno** [reply]
And importantly, it's also addressed with the championed issue, #2235, which would allow the use of a discard in this situation:
```c#
using _ = await ...
```
